src/state/stake/hooks.ts

```diff
@@ -782,20 +779,19 @@ export function useOldSyrupInfo(
           const totalRewardRate = new TokenAmount(token, JSBI.BigInt(rate));
           //const pair = info[index].pair.toLowerCase();
           //const fees = (pairData && pairData[pair] ? pairData[pair].oneDayVolumeUSD * 0.0025: 0);
-          const totalRewardRate01 = new TokenAmount(
+          const rewardRate = new TokenAmount(
```

Comment:
Just to verify the rewards rate never changes as it's the syrup rate and would require redeployment?

Reply:
The reward rate does not change